schemas/dataset-schema.json

```diff
@@ -217,23 +217,38 @@
       "properties": {
         "title": {
           "type": "string",
-          "title": "Title of this ETL dataset",
-          "description": "Title of the dataset (mostly for internal purposes, or for users of our data catalog) which is a one-line description of the dataset (follow the guidelines of the origin's `data_product_title`). NOTE: Dataset titles should be propagated automatically from snapshots. By default, the title of the dataset will be the title of the containing table. But, if there are multiple tables, or if the user wants to manually edit the dataset title, it can be done by editing `dataset.title`.",
-          "requirement_level": "required (often automatic)"
+          "title": "Title of the ETL dataset",
+          "description": "Title of the dataset (mostly for internal purposes, or for users of our data catalog) which is a one-line description of the dataset. NOTE: Dataset titles should be propagated automatically from snapshots. By default, the title of the dataset will be the title of the containing table. But, if there are multiple tables, or if the user wants to manually edit the dataset title, it can be done by editing `dataset.title`.",
```

Comment:
> to manually edit the dataset title, it can be done by editing `dataset.title`.

I find it strange to refer to the field itself with the explicit name. I'd go for

> to manually edit the dataset title, it can be done by editing _this field_.

Reply:
Also, the description seems long enough that it would benefit from some line breaks?

> Title of the dataset (mostly for internal purposes, or for users of our data catalog) which is a one-line description of the dataset.\n\n**Note:** Dataset titles should be propagated automatically from snapshots. By default, the title of the dataset will be the title of the containing table. But, if there are multiple tables, or if the user wants to manually edit the dataset title, it can be done by editing this field.